packages/react-aria-components/src/Menu.tsx

```diff
@@ -191,7 +191,7 @@ function MenuSection<T>({section, className, style, ...otherProps}: MenuSectionP
   );
 }
 
-export interface MenuItemRenderProps extends ItemRenderProps {}
+export interface MenuItemRenderProps extends Omit<ItemRenderProps, 'isInteractive'> {}
```

Comment:
Menu items don't have a non-interactive state other than being disabled unlike Table/GridList/ListBox rows so not including `isInteractive` in the render props